website/source/docs/builders/triton.html.md

```diff
@@ -100,6 +100,24 @@ builder.
     firewall rules. Unless you have a global rule defined in Triton which allows
     SSH traffic enabling the firewall will interfere with the SSH provisioner.
     The default is `false`.
+    
+-   `source_machine_firewall_detail` (object) - Gives the ability to create a temporary
+    firewall rule that allows access to the source machine when `source_machine_firewall_enabled`
+    is enabled. Example:
+    
+    ```json
+    {
```

Comment:
I would likely remove the top level object here in order that this is easily copy/pastable.